cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -295,7 +294,7 @@ public void getAll(@NotNull Context ctx) {
                 if (begin != null) {
                     beginZdt = DateUtils.parseUserDate(begin, timezone);
                 } else {
-                    beginZdt = endZdt.minusHours(24);
+                    beginZdt = ZonedDateTime.of(1900, 1, 1, 0, 0, 0, 0, tz);
```

Comment:
While generally everything practical does start at 1900, it would probably make sense to set this as far back as ZonedDateTime allows. Does ZonedDateTime have an "earliest date" function?

Reply:
Works for me. Thanks.